examples/gno.land/r/samourai/samourai.gno

```diff
@@ -0,0 +1,21 @@
+package samourai
+
```

Comment:
does this work? I'm not sure you can push a package at the namespace root
this should probably be /r/samourai/home and be a proxy render with samdao as admin

Reply:
they have this in r/demo with demo.gno

Reply:
okay, should still be a render proxy managed by the samdao IMO

Reply:
Agree but anyway let's focus on daokit, i will remove anything not directly related to daokit in the PR towards Gno so the important thing are the: ``/p/daokit`` & ``/p/basedao`` & ``/p/daocond`` & ``/p/role_manager`` & ``/r/daokit_demo``